studio/schemas/documents/sections/awards.js

```diff
@@ -0,0 +1,59 @@
+import supportedLanguages from '../../supportedLanguages'
+import { SplitHorizontalIcon } from '@sanity/icons'
+
+const baseLanguage = supportedLanguages.find(l => l.isDefault)
+
+export default {
+  type: 'document',
+  name: 'awards',
+  title: 'Awards',
+  icon: SplitHorizontalIcon,
+  fields: [
+    {
+      name: 'heading',
+      type: 'localeString',
+      title: 'Heading (*)',
+      validation: Rule => Rule.error('Information required.').required()
+    },
+    {
+      name: 'images',
+      type: 'array',
+      title: 'Images (*)',
+      validation: Rule => [
+        Rule.error('Information required.').required(),
+        Rule.min(1).error('Please, select a country.')
+      ],
+      of: [
+        {
+          title: 'Image',
+          type: 'figure'
+        }
+      ]
+    },
+    {
+      name: 'video',
+      type: 'figure',
+      title: 'Award Video',
+      validation: Rule => Rule.error('Information required.').required()
```

Comment:
Since it is a GIF that will be inserted by the user, I think it is a good idea to change the title and add a description saying that the user can display either a GIF or a static image:

      title: 'Award Display',
      description 'Choose between an animated image (GIF) or a static image',